### ISSUE TITLE
Automatic configuration not working on Nx 14.5 with Angular Apps

### ISSUE THREAD

**Issue by nicolashzmor** [wallabyjs/public#3047]
When creating a new workspace with create-nx-workspace for Nest + Angular apps (although this was also checked in a running project with blank workspace), Wallaby cannot run tests on Angular Apps. Not sure if this happens with other frameworks, didn't try.

### Steps to reproduce
1 - Create fresh new workspace
```bash
npx create-nx-workspace --name failing-wallaby --preset angular-nest
```
2 - Run Wallaby
In WebStorm, try running tests from IDE with automatic configuration.

Outcome
```shell
wallaby.js started
core v1.0.1317
Wallaby App (realtime reports) is available at: http://localhost:51245
console.log: Jest: PASS api apps/api/src/app/app.service.spec.ts

console.log: Jest: PASS api apps/api/src/app/app.controller.spec.ts

Some long running code has been detected: one of your files is taking more than 5000ms to execute.
Execution of the following files has started but has not finished:
- apps/test-workspace/src/test-setup.ts
Try commenting out the test or excluding the test file from the `tests` list in your wallaby config,
and restarting wallaby to make sure that it is this test/file causing the issue and not something else.
Pinging test runner sandbox...
The sandbox is not responsive. Check for possibly recently introduced infinite loops.
```
As it could be seen, the API tests are passing without issue.
Angular app doesn't pass throught, as it gets in a loop.

### Other attempts

I had the following sequence of strange events:
1 - I removed the `test-setup.ts` file from the tests flow (commenting the lines inside)
     The error moved the failing file to the only `*.spec.ts` in directory

2 - Removed all tests and placed a `expect(true).toEqual(false)`
     The tests now pass without a problem

3 - Added an empty `TestBed`. Just imported it and put it in the `beforeEach` section.
     The above error is shown again with the `*.spec.ts` file failing.

### Results of running `nx run test-workspace:test`
```

 PASS   test-workspace  apps/test-workspace/src/app/app.component.spec.ts
  AppComponent
    ✓ should create the app (33 ms)

Test Suites: 1 passed, 1 total
Tests:       1 passed, 1 total
Snapshots:   0 total
Time:        3.193 s
Ran all test suites.

 —————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————

 >  NX   Successfully ran target test for project test-workspace (4s)

```

**Comment by smcenlly** [comment]
We tried to reproduce your problem on both Windows and MacOS and it's working for us in both.

Could you please clone [this repo](https://github.com/wallabyjs/wallaby-3047) and see if it's working for you?

If you try and generate your [Wallaby Diagnostics report](https://wallabyjs.com/docs/intro/troubleshooting.html#diagnostics-report) do you get output to your clipboard? If so, can you please reply with the content?

If you're still having issues and the diagnostic report is not returning anything to you, can you please edit your `package.json`, and add:

```
...
  "wallaby": {
    "debug": true,
    "trace", true
  }
...
```

Once you have changed your `package.json`, please start Wallaby again and when it hangs, then provide us with the output of your Wallaby Console from the Run Configuration output.

**Comment by nicolashzmor** [comment]
@smcenlly thanks for your response. Definitely, I have something that's not working with Wallaby.

I clone the repository, configure wallaby in webstorm and it still fails.

These are the last lines of the reported in the console:
```bash
2022-08-16T08:12:14.533Z workers [ycy2v] Run 1 test(s), skipped 0 test(s)
2022-08-16T08:12:14.533Z workers Jest Test Run Complete (ycy2v): 2022-08-16T08:12:14.529Z
2022-08-16T08:12:14.535Z workers [ycy2v] Sandbox is responsive, closing it
Some long running code has been detected: one of your files is taking more than 5000ms to execute.
2022-08-16T08:12:19.527Z workers Some long running code has been detected: one of your files is taking more than 5000ms to execute.
Execution of the following files has started but has not finished:
Execution of the following files has started but has not finished:
- apps/my-app/src/test-setup.ts
- apps/my-app/src/test-setup.ts
Try commenting out the test or excluding the test file from the `tests` list in your wallaby config,
and restarting wallaby to make sure that it is this test/file causing the issue and not something else.
Try commenting out the test or excluding the test file from the `tests` list in your wallaby config,
Pinging test runner sandbox...
and restarting wallaby to make sure that it is this test/file causing the issue and not something else.
Pinging test runner sandbox...
The sandbox is not responsive. Check for possibly recently introduced infinite loops.
2022-08-16T08:12:19.727Z workers The sandbox is not responsive. Check for possibly recently introduced infinite loops.
```

Just mentioning, I am working with `node@16.16.0` and `npm@8.11.0`.

The only thing that I have to change from a completely default configuration is this, as the `{SYSTEM_DEFAULT}` is not working.

![Screenshot from 2022-08-16 10-18-23](https://user-images.githubusercontent.com/66749276/184832331-db2b6e57-edc2-4714-a274-5169a003f6e5.png)

**Comment by smcenlly** [comment]
Could you please tell us what operating system you are using?

**Comment by smcenlly** [comment]
A couple of other follow up questions:

Did you try and generate your [Wallaby Diagnostics report](https://wallabyjs.com/docs/intro/troubleshooting.html#diagnostics-report)? Do you get output to your clipboard? If so, can you please reply with the content?

If you have any anti-virus scanner, can you please try disabling it and see if that helps? It may be incorrectly detecting Wallaby as a false positive and stopping your code from executing.

**Comment by nicolashzmor** [comment]
@smcenlly thanks for your response.

I'm running over ubuntu 22.04 and I have no anti-virus installed.

Regarding the diagnostic report, here it is:

```bash
```
{
  editorVersion: 'WebStorm 2022.1.3',
  pluginVersion: '1.0.251',
  editorType: 'IntelliJ',
  osVersion: 'linux 5.17.0-1015-oem',
  nodeVersion: 'v16.16.0',
  coreVersion: '1.0.1322',
  checksum: 'YmUxNmZkNWYyNWI4NTcwNmUzYmFjMzJmNGUzMzYxNDcsMTY2ODAzODQwMDAwMCww',
  config: {
    debug: true,
    trace: true,
    diagnostics: {
      angular: {
        workspace: {
          '$schema': './node_modules/nx/schemas/nx-schema.json',
          version: 1,
          projects: {
            api: {
              root: 'apps/api',
              '$schema': '../../node_modules/nx/schemas/project-schema.json',
              sourceRoot: 'apps/api/src',
              projectType: 'application',
              architect: {
                build: {
                  builder: '@nrwl/node:webpack',
                  outputs: [ '{options.outputPath}' ],
                  options: { outputPath: 'dist/apps/api', main: 'apps/api/src/main.ts', tsConfig: 'apps/api/tsconfig.app.json', assets: [ 'apps/api/src/assets' ] },
                  configurations: {
                    production: {
                      optimization: true,
                      extractLicenses: true,
                      inspect: false,
                      fileReplacements: [ { replace: 'apps/api/src/environments/environment.ts', with: 'apps/api/src/environments/environment.prod.ts' } ]
                    }
                  },
                  dependsOn: [ '^build' ]
                },
                serve: { builder: '@nrwl/node:node', options: { buildTarget: 'api:build' }, configurations: { production: { buildTarget: 'api:build:production' } } },
                lint: { builder: '@nrwl/linter:eslint', outputs: [ '{options.outputFile}' ], options: { lintFilePatterns: [ 'apps/api/**/*.ts' ] } },
                test: {
                  builder: '@nrwl/jest:jest',
                  outputs: [ 'coverage/apps/api' ],
                  options: { jestConfig: '<homeDir>/Documents/doesrobbiedream/wallaby-3047/apps/api/jest.config.ts', passWithNoTests: true }
                }
              },
              tags: []
            },
            'api-interfaces': {
              root: 'libs/api-interfaces',
              '$schema': '../../node_modules/nx/schemas/project-schema.json',
              sourceRoot: 'libs/api-interfaces/src',
              projectType: 'library',
              architect: {
                lint: { builder: '@nrwl/linter:eslint', outputs: [ '{options.outputFile}' ], options: { lintFilePatterns: [ 'libs/api-interfaces/**/*.ts' ] } }
              },
              tags: []
            },
            'my-app': {
              root: 'apps/my-app',
              projectType: 'application',
              sourceRoot: 'apps/my-app/src',
              prefix: 'failing-wallaby',
              architect: {
                build: {
                  builder: '@angular-devkit/build-angular:browser',
                  outputs: [ '{options.outputPath}' ],
                  options: {
                    outputPath: 'dist/apps/my-app',
                    index: 'apps/my-app/src/index.html',
                    main: 'apps/my-app/src/main.ts',
                    polyfills: 'apps/my-app/src/polyfills.ts',
                    tsConfig: 'apps/my-app/tsconfig.app.json',
                    assets: [ 'apps/my-app/src/favicon.ico', 'apps/my-app/src/assets' ],
                    styles: [ 'apps/my-app/src/styles.css' ],
                    scripts: []
                  },
                  configurations: {
                    production: {
                      budgets: [ { type: 'initial', maximumWarning: '500kb', maximumError: '1mb' }, { type: 'anyComponentStyle', maximumWarning: '2kb', maximumError: '4kb' } ],
                      fileReplacements: [ { replace: 'apps/my-app/src/environments/environment.ts', with: 'apps/my-app/src/environments/environment.prod.ts' } ],
                      outputHashing: 'all'
                    },
                    development: { buildOptimizer: false, optimization: false, vendorChunk: true, extractLicenses: false, sourceMap: true, namedChunks: true }
                  },
                  defaultConfiguration: 'production',
                  dependsOn: [ '^build' ]
                },
                serve: {
                  builder: '@angular-devkit/build-angular:dev-server',
                  configurations: { production: { browserTarget: 'my-app:build:production' }, development: { browserTarget: 'my-app:build:development' } },
                  defaultConfiguration: 'development',
                  options: { proxyConfig: 'apps/my-app/proxy.conf.json' }
                },
                'extract-i18n': { builder: '@angular-devkit/build-angular:extract-i18n', options: { browserTarget: 'my-app:build' } },
                lint: { builder: '@nrwl/linter:eslint', options: { lintFilePatterns: [ 'apps/my-app/**/*.ts', 'apps/my-app/**/*.html' ] } },
                test: {
                  builder: '@nrwl/jest:jest',
                  outputs: [ 'coverage/apps/my-app' ],
                  options: { jestConfig: '<homeDir>/Documents/doesrobbiedream/wallaby-3047/apps/my-app/jest.config.ts', passWithNoTests: true }
                }
              },
              tags: []
            },
            'my-app-e2e': {
              root: 'apps/my-app-e2e',
              '$schema': '../../node_modules/nx/schemas/project-schema.json',
              sourceRoot: 'apps/my-app-e2e/src',
              projectType: 'application',
              architect: {
                e2e: {
                  builder: '@nrwl/cypress:cypress',
                  options: { cypressConfig: 'apps/my-app-e2e/cypress.config.ts', devServerTarget: 'my-app:serve:development', testingType: 'e2e' },
                  configurations: { production: { devServerTarget: 'my-app:serve:production' } }
                },
                lint: { builder: '@nrwl/linter:eslint', outputs: [ '{options.outputFile}' ], options: { lintFilePatterns: [ 'apps/my-app-e2e/**/*.{js,ts}' ] } }
              },
              tags: [],
              implicitDependencies: [ 'my-app' ]
            }
          },
          npmScope: 'failing-wallaby',
          affected: { defaultBase: 'main' },
          cli: { packageManager: 'npm' },
          implicitDependencies: { 'package.json': { dependencies: '*', devDependencies: '*' }, '.eslintrc.json': '*' },
          tasksRunnerOptions: {
            default: { runner: 'nx/tasks-runners/default', options: { cacheableOperations: [ 'build', 'lint', 'test', 'e2e' ] } }
          },
          targetDefaults: { build: { dependsOn: [ '^build' ] } },
          schematics: {
            '@nrwl/angular:application': { style: 'css', linter: 'eslint', unitTestRunner: 'jest', e2eTestRunner: 'cypress' },
            '@nrwl/angular:library': { linter: 'eslint', unitTestRunner: 'jest' },
            '@nrwl/angular:component': { style: 'css' }
          },
          defaultProject: 'my-app',
          configFileName: '<homeDir>/Documents/doesrobbiedream/wallaby-3047/angular.json',
          nxWorkspace: true
        }
      },
      jest: {
        config: {
          configs: [
            {
              automock: false,
              cache: true,
              cacheDirectory: '/tmp/jest_rt',
              clearMocks: false,
              coveragePathIgnorePatterns: [ '/node_modules/' ],
              cwd: '<homeDir>/Documents/doesrobbiedream/wallaby-3047',
              dependencyExtractor: undefined,
              detectLeaks: false,
              detectOpenHandles: false,
              displayName: { color: 'white', name: 'api' },
              errorOnDeprecated: false,
              extensionsToTreatAsEsm: [],
              extraGlobals: [],
              filter: undefined,
              forceCoverageMatch: [],
              globalSetup: undefined,
              globalTeardown: undefined,
              globals: { 'ts-jest': { tsconfig: '<rootDir>/tsconfig.spec.json' } },
              haste: { computeSha1: false, enableSymlinks: false, forceNodeFilesystemAPI: false, throwOnModuleCollision: false },
              injectGlobals: true,
              moduleDirectories: [ 'node_modules' ],
              moduleFileExtensions: [ 'ts', 'js', 'html' ],
              moduleLoader: undefined,
              moduleNameMapper: [],
              modulePathIgnorePatterns: [],
              modulePaths: undefined,
              name: 'b1fef0cfc144b879ece541bcf4e8dac2',
              prettierPath: 'prettier',
              resetMocks: false,
              resetModules: false,
              resolver: '<homeDir>/Documents/doesrobbiedream/wallaby-3047/node_modules/@nrwl/jest/plugins/resolver.js',
              restoreMocks: false,
              rootDir: '<homeDir>/Documents/doesrobbiedream/wallaby-3047/apps/api',
              roots: [ '<homeDir>/Documents/doesrobbiedream/wallaby-3047/apps/api' ],
              runner: '<homeDir>/Documents/doesrobbiedream/wallaby-3047/node_modules/jest-runner/build/index.js',
              setupFiles: [],
              setupFilesAfterEnv: [],
              skipFilter: false,
              skipNodeResolution: undefined,
              slowTestThreshold: 5,
              snapshotFormat: undefined,
              snapshotResolver: undefined,
              snapshotSerializers: [],
              testEnvironment: '<homeDir>/Documents/doesrobbiedream/wallaby-3047/node_modules/jest-environment-node/build/index.js',
              testEnvironmentOptions: {},
              testLocationInResults: false,
              testMatch: [ '**/+(*.)+(spec|test).+(ts|js)?(x)' ],
              testPathIgnorePatterns: [ '/node_modules/' ],
              testRegex: [],
              testRunner: '<homeDir>/Documents/doesrobbiedream/wallaby-3047/node_modules/jest-circus/runner.js',
              testURL: 'http://localhost',
              timers: 'real',
              transform: [
                [ '^.+\\.[tj]s$', '<homeDir>/Documents/doesrobbiedream/wallaby-3047/node_modules/ts-jest/dist/index.js', {} ],
                [ '^.+\\.(ts|js|html)$', '<homeDir>/Documents/doesrobbiedream/wallaby-3047/node_modules/ts-jest/dist/index.js', {} ]
              ],
              transformIgnorePatterns: [ '/node_modules/', '\\.pnp\\.[^\\/]+$' ],
              unmockedModulePathPatterns: undefined,
              watchPathIgnorePatterns: []
            },
            {
              automock: false,
              cache: true,
              cacheDirectory: '/tmp/jest_rt',
              clearMocks: false,
              coveragePathIgnorePatterns: [ '/node_modules/' ],
              cwd: '<homeDir>/Documents/doesrobbiedream/wallaby-3047',
              dependencyExtractor: undefined,
              detectLeaks: false,
              detectOpenHandles: false,
              displayName: { color: 'white', name: 'my-app' },
              errorOnDeprecated: false,
              extensionsToTreatAsEsm: [],
              extraGlobals: [],
              filter: undefined,
              forceCoverageMatch: [],
              globalSetup: undefined,
              globalTeardown: undefined,
              globals: { 'ts-jest': { tsconfig: '<rootDir>/tsconfig.spec.json', stringifyContentPathRegex: '\\.(html|svg)$' } },
              haste: { computeSha1: false, enableSymlinks: false, forceNodeFilesystemAPI: false, throwOnModuleCollision: false },
              injectGlobals: true,
              moduleDirectories: [ 'node_modules' ],
              moduleFileExtensions: [ 'ts', 'js', 'mjs', 'html' ],
              moduleLoader: undefined,
              moduleNameMapper: [],
              modulePathIgnorePatterns: [],
              modulePaths: undefined,
              name: '8f1a66aaa0e6674c678aaffc00da5636',
              prettierPath: 'prettier',
              resetMocks: false,
              resetModules: false,
              resolver: '<homeDir>/Documents/doesrobbiedream/wallaby-3047/node_modules/@nrwl/jest/plugins/resolver.js',
              restoreMocks: false,
              rootDir: '<homeDir>/Documents/doesrobbiedream/wallaby-3047/apps/my-app',
              roots: [ '<homeDir>/Documents/doesrobbiedream/wallaby-3047/apps/my-app' ],
              runner: '<homeDir>/Documents/doesrobbiedream/wallaby-3047/node_modules/jest-runner/build/index.js',
              setupFiles: [],
              setupFilesAfterEnv: [ '<homeDir>/Documents/doesrobbiedream/wallaby-3047/apps/my-app/src/test-setup.ts' ],
              skipFilter: false,
              skipNodeResolution: undefined,
              slowTestThreshold: 5,
              snapshotFormat: undefined,
              snapshotResolver: undefined,
              snapshotSerializers: [
                '<homeDir>/Documents/doesrobbiedream/wallaby-3047/node_modules/jest-preset-angular/build/serializers/no-ng-attributes.js',
                '<homeDir>/Documents/doesrobbiedream/wallaby-3047/node_modules/jest-preset-angular/build/serializers/ng-snapshot.js',
                '<homeDir>/Documents/doesrobbiedream/wallaby-3047/node_modules/jest-preset-angular/build/serializers/html-comment.js'
              ],
              testEnvironment: '<homeDir>/Documents/doesrobbiedream/wallaby-3047/node_modules/jest-environment-jsdom/build/index.js',
              testEnvironmentOptions: {},
              testLocationInResults: false,
              testMatch: [ '**/+(*.)+(spec|test).+(ts|js)?(x)' ],
              testPathIgnorePatterns: [ '/node_modules/' ],
              testRegex: [],
              testRunner: '<homeDir>/Documents/doesrobbiedream/wallaby-3047/node_modules/jest-circus/runner.js',
              testURL: 'http://localhost',
              timers: 'real',
              transform: [
                [ '^.+\\.(ts|mjs|js|html)$', '<homeDir>/Documents/doesrobbiedream/wallaby-3047/node_modules/jest-preset-angular/build/index.js', {} ],
                [ '^.+\\.(ts|js|html)$', '<homeDir>/Documents/doesrobbiedream/wallaby-3047/node_modules/ts-jest/dist/index.js', {} ]
              ],
              transformIgnorePatterns: [ 'node_modules/(?!.*\\.mjs$)' ],
              unmockedModulePathPatterns: undefined,
              watchPathIgnorePatterns: []
            }
          ],
          globalConfig: {
            bail: 0,
            changedFilesWithAncestor: false,
            changedSince: undefined,
            collectCoverage: false,
            collectCoverageFrom: [],
            collectCoverageOnlyFrom: undefined,
            coverageDirectory: '<homeDir>/Documents/doesrobbiedream/wallaby-3047/coverage/apps/api',
            coverageProvider: 'babel',
            coverageReporters: [ 'html' ],
            coverageThreshold: undefined,
            detectLeaks: false,
            detectOpenHandles: false,
            errorOnDeprecated: false,
            expand: false,
            filter: undefined,
            findRelatedTests: false,
            forceExit: false,
            globalSetup: undefined,
            globalTeardown: undefined,
            json: false,
            lastCommit: false,
            listTests: false,
            logHeapUsage: false,
            maxConcurrency: 5,
            maxWorkers: 15,
            noSCM: undefined,
            noStackTrace: false,
            nonFlagArgs: [],
            notify: false,
            notifyMode: 'failure-change',
            onlyChanged: false,
            onlyFailures: false,
            outputFile: undefined,
            passWithNoTests: true,
            projects: [],
            replname: undefined,
            reporters: undefined,
            rootDir: '<homeDir>/Documents/doesrobbiedream/wallaby-3047/apps/api',
            runTestsByPath: false,
            silent: undefined,
            skipFilter: false,
            snapshotFormat: undefined,
            testFailureExitCode: 1,
            testNamePattern: undefined,
            testPathPattern: '',
            testResultsProcessor: undefined,
            testSequencer: '<homeDir>/Documents/doesrobbiedream/wallaby-3047/node_modules/@jest/test-sequencer/build/index.js',
            testTimeout: undefined,
            updateSnapshot: 'new',
            useStderr: false,
            verbose: undefined,
            watch: false,
            watchAll: false,
            watchPlugins: undefined,
            watchman: true
          },
          hasDeprecationWarnings: false,
          wallaby: {
            roots: [ 'apps/api', 'apps/my-app' ],
            watchPathIgnorePatterns: [ '/node_modules/', '\\./dist/|\\./build/|\\./coverage/|\\./git/|/\\..+/', '/tmp/jest_rt', '\\./coverage/apps/api' ],
            testPathIgnorePatterns: [ '/node_modules/', '\\./dist/|\\./build/|\\./coverage/|\\./git/|/\\..+/', '/tmp/jest_rt', '\\./coverage/apps/api' ],
            testMatch: [ '**/+(*.)+(spec|test).+(ts|js)?(x)' ],
            testRegex: []
          }
        }
      }
    },
    testFramework: { version: 'jest@24.8.0', configurator: 'jest@24.8.0', reporter: 'jest@24.8.0', starter: 'jest@24.8.0', autoDetected: true },
    filesWithCoverageCalculated: [],
    filesWithNoCoverageCalculated: [],
    globalSetup: false,
    dot: true,
    customJestConfig: {
      configs: [
        {
          automock: false,
          cache: true,
          cacheDirectory: '/tmp/jest_rt',
          clearMocks: false,
          coveragePathIgnorePatterns: [ '/node_modules/' ],
          cwd: '<homeDir>/Documents/doesrobbiedream/wallaby-3047',
          dependencyExtractor: undefined,
          detectLeaks: false,
          detectOpenHandles: false,
          displayName: { color: 'white', name: 'api' },
          errorOnDeprecated: false,
          extensionsToTreatAsEsm: [],
          extraGlobals: [],
          filter: undefined,
          forceCoverageMatch: [],
          globalSetup: undefined,
          globalTeardown: undefined,
          globals: { 'ts-jest': { tsconfig: '<rootDir>/tsconfig.spec.json' } },
          haste: { computeSha1: false, enableSymlinks: false, forceNodeFilesystemAPI: false, throwOnModuleCollision: false },
          injectGlobals: true,
          moduleDirectories: [ 'node_modules' ],
          moduleFileExtensions: [ 'ts', 'js', 'html' ],
          moduleLoader: undefined,
          moduleNameMapper: [],
          modulePathIgnorePatterns: [],
          modulePaths: undefined,
          name: 'b1fef0cfc144b879ece541bcf4e8dac2',
          prettierPath: 'prettier',
          resetMocks: false,
          resetModules: false,
          resolver: '<homeDir>/Documents/doesrobbiedream/wallaby-3047/node_modules/@nrwl/jest/plugins/resolver.js',
          restoreMocks: false,
          rootDir: '<homeDir>/Documents/doesrobbiedream/wallaby-3047/apps/api',
          roots: [ '<homeDir>/Documents/doesrobbiedream/wallaby-3047/apps/api' ],
          runner: '<homeDir>/Documents/doesrobbiedream/wallaby-3047/node_modules/jest-runner/build/index.js',
          setupFiles: [],
          setupFilesAfterEnv: [],
          skipFilter: false,
          skipNodeResolution: undefined,
          slowTestThreshold: 5,
          snapshotFormat: undefined,
          snapshotResolver: undefined,
          snapshotSerializers: [],
          testEnvironment: '<homeDir>/Documents/doesrobbiedream/wallaby-3047/node_modules/jest-environment-node/build/index.js',
          testEnvironmentOptions: {},
          testLocationInResults: false,
          testMatch: [ '**/+(*.)+(spec|test).+(ts|js)?(x)' ],
          testPathIgnorePatterns: [ '/node_modules/' ],
          testRegex: [],
          testRunner: '<homeDir>/Documents/doesrobbiedream/wallaby-3047/node_modules/jest-circus/runner.js',
          testURL: 'http://localhost',
          timers: 'real',
          transform: [
            [ '^.+\\.[tj]s$', '<homeDir>/Documents/doesrobbiedream/wallaby-3047/node_modules/ts-jest/dist/index.js', {} ],
            [ '^.+\\.(ts|js|html)$', '<homeDir>/Documents/doesrobbiedream/wallaby-3047/node_modules/ts-jest/dist/index.js', {} ]
          ],
          transformIgnorePatterns: [ '/node_modules/', '\\.pnp\\.[^\\/]+$' ],
          unmockedModulePathPatterns: undefined,
          watchPathIgnorePatterns: []
        },
        {
          automock: false,
          cache: true,
          cacheDirectory: '/tmp/jest_rt',
          clearMocks: false,
          coveragePathIgnorePatterns: [ '/node_modules/' ],
          cwd: '<homeDir>/Documents/doesrobbiedream/wallaby-3047',
          dependencyExtractor: undefined,
          detectLeaks: false,
          detectOpenHandles: false,
          displayName: { color: 'white', name: 'my-app' },
          errorOnDeprecated: false,
          extensionsToTreatAsEsm: [],
          extraGlobals: [],
          filter: undefined,
          forceCoverageMatch: [],
          globalSetup: undefined,
          globalTeardown: undefined,
          globals: { 'ts-jest': { tsconfig: '<rootDir>/tsconfig.spec.json', stringifyContentPathRegex: '\\.(html|svg)$' } },
          haste: { computeSha1: false, enableSymlinks: false, forceNodeFilesystemAPI: false, throwOnModuleCollision: false },
          injectGlobals: true,
          moduleDirectories: [ 'node_modules' ],
          moduleFileExtensions: [ 'ts', 'js', 'mjs', 'html' ],
          moduleLoader: undefined,
          moduleNameMapper: [],
          modulePathIgnorePatterns: [],
          modulePaths: undefined,
          name: '8f1a66aaa0e6674c678aaffc00da5636',
          prettierPath: 'prettier',
          resetMocks: false,
          resetModules: false,
          resolver: '<homeDir>/Documents/doesrobbiedream/wallaby-3047/node_modules/@nrwl/jest/plugins/resolver.js',
          restoreMocks: false,
          rootDir: '<homeDir>/Documents/doesrobbiedream/wallaby-3047/apps/my-app',
          roots: [ '<homeDir>/Documents/doesrobbiedream/wallaby-3047/apps/my-app' ],
          runner: '<homeDir>/Documents/doesrobbiedream/wallaby-3047/node_modules/jest-runner/build/index.js',
          setupFiles: [],
          setupFilesAfterEnv: [ '<homeDir>/Documents/doesrobbiedream/wallaby-3047/apps/my-app/src/test-setup.ts' ],
          skipFilter: false,
          skipNodeResolution: undefined,
          slowTestThreshold: 5,
          snapshotFormat: undefined,
          snapshotResolver: undefined,
          snapshotSerializers: [
            '<homeDir>/Documents/doesrobbiedream/wallaby-3047/node_modules/jest-preset-angular/build/serializers/no-ng-attributes.js',
            '<homeDir>/Documents/doesrobbiedream/wallaby-3047/node_modules/jest-preset-angular/build/serializers/ng-snapshot.js',
            '<homeDir>/Documents/doesrobbiedream/wallaby-3047/node_modules/jest-preset-angular/build/serializers/html-comment.js'
          ],
          testEnvironment: '<homeDir>/Documents/doesrobbiedream/wallaby-3047/node_modules/jest-environment-jsdom/build/index.js',
          testEnvironmentOptions: {},
          testLocationInResults: false,
          testMatch: [ '**/+(*.)+(spec|test).+(ts|js)?(x)' ],
          testPathIgnorePatterns: [ '/node_modules/' ],
          testRegex: [],
          testRunner: '<homeDir>/Documents/doesrobbiedream/wallaby-3047/node_modules/jest-circus/runner.js',
          testURL: 'http://localhost',
          timers: 'real',
          transform: [
            [ '^.+\\.(ts|mjs|js|html)$', '<homeDir>/Documents/doesrobbiedream/wallaby-3047/node_modules/jest-preset-angular/build/index.js', {} ],
            [ '^.+\\.(ts|js|html)$', '<homeDir>/Documents/doesrobbiedream/wallaby-3047/node_modules/ts-jest/dist/index.js', {} ]
          ],
          transformIgnorePatterns: [ 'node_modules/(?!.*\\.mjs$)' ],
          unmockedModulePathPatterns: undefined,
          watchPathIgnorePatterns: []
        }
      ],
      globalConfig: {
        bail: 0,
        changedFilesWithAncestor: false,
        changedSince: undefined,
        collectCoverage: false,
        collectCoverageFrom: [],
        collectCoverageOnlyFrom: undefined,
        coverageDirectory: '<homeDir>/Documents/doesrobbiedream/wallaby-3047/coverage/apps/api',
        coverageProvider: 'babel',
        coverageReporters: [ 'html' ],
        coverageThreshold: undefined,
        detectLeaks: false,
        detectOpenHandles: false,
        errorOnDeprecated: false,
        expand: false,
        filter: undefined,
        findRelatedTests: false,
        forceExit: false,
        globalSetup: undefined,
        globalTeardown: undefined,
        json: false,
        lastCommit: false,
        listTests: false,
        logHeapUsage: false,
        maxConcurrency: 5,
        maxWorkers: 15,
        noSCM: undefined,
        noStackTrace: false,
        nonFlagArgs: [],
        notify: false,
        notifyMode: 'failure-change',
        onlyChanged: false,
        onlyFailures: false,
        outputFile: undefined,
        passWithNoTests: true,
        projects: [],
        replname: undefined,
        reporters: undefined,
        rootDir: '<homeDir>/Documents/doesrobbiedream/wallaby-3047/apps/api',
        runTestsByPath: false,
        silent: undefined,
        skipFilter: false,
        snapshotFormat: undefined,
        testFailureExitCode: 1,
        testNamePattern: undefined,
        testPathPattern: '',
        testResultsProcessor: undefined,
        testSequencer: '<homeDir>/Documents/doesrobbiedream/wallaby-3047/node_modules/@jest/test-sequencer/build/index.js',
        testTimeout: undefined,
        updateSnapshot: 'new',
        useStderr: false,
        verbose: undefined,
        watch: false,
        watchAll: false,
        watchPlugins: undefined,
        watchman: true
      },
      hasDeprecationWarnings: false,
      wallaby: {
        roots: [ 'apps/api', 'apps/my-app' ],
        watchPathIgnorePatterns: [ '/node_modules/', '\\./dist/|\\./build/|\\./coverage/|\\./git/|/\\..+/', '/tmp/jest_rt', '\\./coverage/apps/api' ],
        testPathIgnorePatterns: [ '/node_modules/', '\\./dist/|\\./build/|\\./coverage/|\\./git/|/\\..+/', '/tmp/jest_rt', '\\./coverage/apps/api' ],
        testMatch: [ '**/+(*.)+(spec|test).+(ts|js)?(x)' ],
        testRegex: []
      }
    },
    files: [
      { pattern: '/node_modules/', regexp: /\/node_modules\//, ignore: true, trigger: true, load: true, file: true, test: true },
      {
        pattern: '\\./dist/|\\./build/|\\./coverage/|\\./git/|/\\..+/',
        regexp: /\.\/dist\/|\.\/build\/|\.\/coverage\/|\.\/git\/|\/\..+\//,
        ignore: true,
        trigger: true,
        load: true,
        file: true,
        test: true
      },
      { pattern: '/tmp/jest_rt', regexp: /\/tmp\/jest_rt/, ignore: true, trigger: true, load: true, file: true, test: true },
      { pattern: '\\./coverage/apps/api', regexp: /\.\/coverage\/apps\/api/, ignore: true, trigger: true, load: true, file: true, test: true },
      { pattern: '**/**', ignore: false, trigger: true, load: true, order: 1 },
      { pattern: 'apps/api/**', ignore: false, trigger: true, load: true, order: 2 },
      { pattern: 'apps/my-app/**', ignore: false, trigger: true, load: true, order: 3 },
      { pattern: 'apps/api/**/+(*.)+(spec|test).+(ts|js)?(x)', ignore: true, trigger: true, load: true, file: true },
      { pattern: 'apps/my-app/**/+(*.)+(spec|test).+(ts|js)?(x)', ignore: true, trigger: true, load: true, file: true }
    ],
    tests: [
      { pattern: '/node_modules/', regexp: /\/node_modules\//, ignore: true, trigger: true, load: true, test: true, file: false },
      {
        pattern: '\\./dist/|\\./build/|\\./coverage/|\\./git/|/\\..+/',
        regexp: /\.\/dist\/|\.\/build\/|\.\/coverage\/|\.\/git\/|\/\..+\//,
        ignore: true,
        trigger: true,
        load: true,
        test: true,
        file: false
      },
      { pattern: '/tmp/jest_rt', regexp: /\/tmp\/jest_rt/, ignore: true, trigger: true, load: true, test: true, file: false },
      { pattern: '\\./coverage/apps/api', regexp: /\.\/coverage\/apps\/api/, ignore: true, trigger: true, load: true, test: true, file: false },
      { pattern: 'apps/api/**/+(*.)+(spec|test).+(ts|js)?(x)', ignore: false, trigger: true, load: true, test: true, order: 4 },
      { pattern: 'apps/my-app/**/+(*.)+(spec|test).+(ts|js)?(x)', ignore: false, trigger: true, load: true, test: true, order: 5 }
    ],
    runAllTestsInAffectedTestFile: false,
    updateNoMoreThanOneSnapshotPerTestFileRun: false,
    compilers: {},
    logLimits: { inline: { depth: 5, elements: 5000 }, values: { default: { stringLength: 200 }, autoExpand: { elements: 5000, stringLength: 8192, depth: 10 } } },
    preprocessors: {},
    maxConsoleMessagesPerTest: 100,
    autoConsoleLog: true,
    delays: { run: 0, edit: 100, update: 0 },
    workers: { initial: 0, regular: 0, recycle: false },
    teardown: undefined,
    hints: {
      ignoreCoverage: '__REGEXP /ignore coverage|istanbul ignore/',
      ignoreCoverageForFile: '__REGEXP /ignore file coverage/',
      commentAutoLog: '?',
      testFileSelection: { include: '__REGEXP /file\\.only/', exclude: '__REGEXP /file\\.skip/' }
    },
    automaticTestFileSelection: true,
    runSelectedTestsOnly: false,
    mapConsoleMessagesStackTrace: false,
    extensions: {},
    env: {
      type: 'node',
      params: {},
      runner: '<homeDir>/.nvm/versions/node/v16.16.0/bin/node',
      viewportSize: { width: 800, height: 600 },
      options: { width: 800, height: 600 },
      bundle: true
    },
    reportUnhandledPromises: true,
    slowTestThreshold: 75,
    lowCoverageThreshold: 80,
    loose: true,
    configCode: 'auto.detect#-627968982'
  },
  packageJSON: {
    dependencies: {
      '@angular/animations': '~14.1.1',
      '@angular/common': '~14.1.1',
      '@angular/compiler': '~14.1.1',
      '@angular/core': '~14.1.1',
      '@angular/forms': '~14.1.1',
      '@angular/platform-browser': '~14.1.1',
      '@angular/platform-browser-dynamic': '~14.1.1',
      '@angular/router': '~14.1.1',
      '@nestjs/common': '^9.0.0',
      '@nestjs/core': '^9.0.0',
      '@nestjs/platform-express': '^9.0.0',
      '@nrwl/angular': '14.5.6',
      'reflect-metadata': '^0.1.13',
      rxjs: '~7.4.0',
      tslib: '^2.3.0',
      'zone.js': '~0.11.4'
    },
    devDependencies: {
      '@angular-devkit/build-angular': '~14.1.1',
      '@angular-eslint/eslint-plugin': '~14.0.0',
      '@angular-eslint/eslint-plugin-template': '~14.0.0',
      '@angular-eslint/template-parser': '~14.0.0',
      '@angular/cli': '~14.1.0',
      '@angular/compiler-cli': '~14.1.1',
      '@angular/language-service': '~14.1.1',
      '@nestjs/schematics': '^9.0.0',
      '@nestjs/testing': '^9.0.0',
      '@nrwl/cli': '14.5.6',
      '@nrwl/cypress': '14.5.6',
      '@nrwl/eslint-plugin-nx': '14.5.6',
      '@nrwl/jest': '14.5.6',
      '@nrwl/linter': '14.5.6',
      '@nrwl/nest': '14.5.6',
      '@nrwl/node': '14.5.6',
      '@nrwl/nx-cloud': 'latest',
      '@nrwl/workspace': '14.5.6',
      '@types/jest': '27.4.1',
      '@types/node': '16.11.7',
      '@typescript-eslint/eslint-plugin': '^5.29.0',
      '@typescript-eslint/parser': '^5.29.0',
      cypress: '^10.2.0',
      eslint: '~8.15.0',
      'eslint-config-prettier': '8.1.0',
      'eslint-plugin-cypress': '^2.10.3',
      jest: '27.5.1',
      'jest-preset-angular': '~11.1.2',
      nx: '14.5.6',
      prettier: '^2.6.2',
      'ts-jest': '27.1.4',
      'ts-node': '10.9.1',
      typescript: '~4.7.2'
    }
  },
  fs: { numberOfFiles: 75 },
  debug: [
    '2022-08-17T07:46:59.133Z config Attempting automatic configuration for angular\n',
    '2022-08-17T07:46:59.234Z angular/cli config Detected Angular CLI.\n',
    '2022-08-17T07:46:59.237Z angular/cli config Angular currentDirPath: "<homeDir>/Documents/doesrobbiedream/wallaby-3047", workspaceDirPath: "<homeDir>/Documents/doesrobbiedream/wallaby-3047"\n',
    '2022-08-17T07:47:00.665Z jest/config Detected Jest.\n',
    '2022-08-17T07:47:00.666Z jest/config Configured Jest.\n',
    '2022-08-17T07:47:00.666Z config Finished attempting automatic configuration for angular (1534ms)\n',
    '2022-08-17T07:47:00.669Z project Wallaby Node version: v16.16.0\n',
    '2022-08-17T07:47:00.669Z project Wallaby config: <homeDir>/Documents/doesrobbiedream/wallaby-3047/auto.detect\n',
    '2022-08-17T07:47:00.680Z fs File system starting\n',
    '2022-08-17T07:47:00.724Z fs File system scan completed\n',
    '2022-08-17T07:47:00.729Z project File cache: <homeDir>/.cache/JetBrains/WebStorm2022.1/wallaby/projects/a4ea19d608d3bc39\n',
    '2022-08-17T07:47:00.765Z uiService Listening port 51235\n',
    '2022-08-17T07:47:00.776Z workers Parallelism for initial run: 14, for regular run: 7\n',
    '2022-08-17T07:47:00.776Z workers Starting run worker instance #0\n',
    '2022-08-17T07:47:00.776Z workers Starting run worker instance #1\n',
    '2022-08-17T07:47:00.776Z workers Starting run worker instance #2\n',
    '2022-08-17T07:47:00.776Z workers Starting run worker instance #3\n',
    '2022-08-17T07:47:00.776Z workers Starting run worker instance #4\n',
    '2022-08-17T07:47:00.776Z workers Starting run worker instance #5\n',
    '2022-08-17T07:47:00.776Z workers Starting run worker instance #6\n',
    '2022-08-17T07:47:00.776Z workers Starting run worker instance #7\n',
    '2022-08-17T07:47:00.776Z workers Starting run worker instance #8\n',
    '2022-08-17T07:47:00.776Z workers Starting run worker instance #9\n',
    '2022-08-17T07:47:00.776Z workers Starting run worker instance #10\n',
    '2022-08-17T07:47:00.776Z workers Starting run worker instance #11\n',
    '2022-08-17T07:47:00.776Z workers Starting run worker instance #12\n',
    '2022-08-17T07:47:00.776Z workers Starting run worker instance #13\n',
    '2022-08-17T07:47:00.777Z workers Web server is listening at 38797\n',
    '2022-08-17T07:47:00.997Z project Stopping process pool\n',
    '2022-08-17T07:47:00.997Z project File cache is up-to-date, starting full test run\n',
    '2022-08-17T07:47:01.000Z project Test run started; run priority: 3\n',
    '2022-08-17T07:47:01.002Z project Running all tests\n',
    '2022-08-17T07:47:01.003Z workers Starting test run, priority: 3\n',
    '2022-08-17T07:47:01.004Z workers Distributing tests between 14 workers\n',
    '2022-08-17T07:47:01.004Z workers Running tests in parallel\n',
    '2022-08-17T07:47:01.004Z nodeRunner Starting sandbox [worker #0, session #toozo]\n',
    '2022-08-17T07:47:01.004Z nodeRunner Starting sandbox [worker #1, session #fdq3y]\n',
    '2022-08-17T07:47:01.004Z nodeRunner Starting sandbox [worker #2, session #wcvpe]\n',
    '2022-08-17T07:47:01.004Z nodeRunner Preparing sandbox [worker #0, session #toozo]\n',
    '2022-08-17T07:47:01.004Z nodeRunner Preparing sandbox [worker #1, session #fdq3y]\n',
    '2022-08-17T07:47:01.004Z nodeRunner Preparing sandbox [worker #2, session #wcvpe]\n',
    '2022-08-17T07:47:01.017Z workers Started run worker instance (delayed) #2\n',
    '2022-08-17T07:47:01.018Z nodeRunner Prepared sandbox [worker #2, session #wcvpe]\n',
    '2022-08-17T07:47:01.018Z workers [worker #2, session #wcvpe] Running tests in sandbox\n',
    '2022-08-17T07:47:01.021Z workers Started run worker instance (delayed) #1\n',
    '2022-08-17T07:47:01.022Z nodeRunner Prepared sandbox [worker #1, session #fdq3y]\n',
    '2022-08-17T07:47:01.022Z workers [worker #1, session #fdq3y] Running tests in sandbox\n',
    '2022-08-17T07:47:01.022Z workers Started run worker instance (delayed) #4\n',
    '2022-08-17T07:47:01.024Z workers Started run worker instance (delayed) #0\n',
    '2022-08-17T07:47:01.024Z nodeRunner Prepared sandbox [worker #0, session #toozo]\n',
    '2022-08-17T07:47:01.024Z workers [worker #0, session #toozo] Running tests in sandbox\n',
    '2022-08-17T07:47:01.029Z workers Started run worker instance (delayed) #3\n',
    '2022-08-17T07:47:01.030Z workers Started run worker instance (delayed) #5\n',
    '2022-08-17T07:47:01.031Z workers Started run worker instance (delayed) #6\n',
    '2022-08-17T07:47:01.063Z workers Started run worker instance (delayed) #7\n',
    '2022-08-17T07:47:01.074Z workers Started run worker instance (delayed) #8\n',
    '2022-08-17T07:47:01.099Z workers Started run worker instance (delayed) #9\n',
    '2022-08-17T07:47:01.109Z workers Started run worker instance (delayed) #10\n',
    '2022-08-17T07:47:01.127Z workers Started run worker instance (delayed) #11\n',
    '2022-08-17T07:47:01.147Z workers Started run worker instance (delayed) #12\n',
    '2022-08-17T07:47:01.161Z workers Started run worker instance (delayed) #13\n',
    '2022-08-17T07:47:04.222Z workers Scheduling Jest Test Run (toozo): 2022-08-17T07:47:02.727Z\n',
    '2022-08-17T07:47:04.228Z workers [toozo] Loaded unknown number of test(s)\n',
    '2022-08-17T07:47:04.228Z workers [toozo] Test executed: should return "Welcome to api!"\n',
    '2022-08-17T07:47:04.229Z workers [toozo] Run 1 test(s), skipped 0 test(s)\n',
    '2022-08-17T07:47:04.229Z workers Jest Test Run Complete (toozo): 2022-08-17T07:47:04.221Z\n',
    '2022-08-17T07:47:04.230Z workers [toozo] Sandbox is responsive, closing it\n',
    '2022-08-17T07:47:04.247Z workers Scheduling Jest Test Run (fdq3y): 2022-08-17T07:47:02.730Z\n',
    '2022-08-17T07:47:04.249Z workers [fdq3y] Loaded unknown number of test(s)\n',
    '2022-08-17T07:47:04.249Z workers [fdq3y] Test executed: should return "Welcome to api!"\n',
    '2022-08-17T07:47:04.250Z workers [fdq3y] Run 1 test(s), skipped 0 test(s)\n',
    '2022-08-17T07:47:04.250Z workers Jest Test Run Complete (fdq3y): 2022-08-17T07:47:04.246Z\n',
    '2022-08-17T07:47:04.251Z workers [fdq3y] Sandbox is responsive, closing it\n',
    '2022-08-17T07:47:08.921Z workers Some long running code has been detected: one of your files is taking more than 5000ms to execute.\n' +
      'Execution of the following files has started but has not finished:\n' +
      '- apps/my-app/src/test-setup.ts\n' +
      'Try commenting out the test or excluding the test file from the `tests` list in your wallaby config,\n' +
      'and restarting wallaby to make sure that it is this test/file causing the issue and not something else.\n' +
      'Pinging test runner sandbox...\n',
    '2022-08-17T07:47:09.120Z workers The sandbox is not responsive. Check for possibly recently introduced infinite loops.\n'
  ]
}
```
```

**Comment by smcenlly** [comment]
> I'm running over ubuntu 22.04 and I have no anti-virus installed.

How did you install `node` and `WebStorm`?

We have had some issues in the past when Ubuntu users have install `node` using the `snapcraft`. There can sometimes be some sandboxing added when node is installed as a `snap`, which stops inter-process communication, which Wallaby uses at runtime.

Could you please try uninstalling `node` and reinstall from [https://nodejs.org/en/](https://nodejs.org/en/) to see if that fixes your problem?

**Comment by nicolashzmor** [comment]
I haven't used Snapcraft. I'm installing via nvm, which I have used before without problems...

I'll try anyway to install a plain version to see if that fixes it.

Thanks again for your responses,

**Comment by nicolashzmor** [comment]
So far:

- Removed Node and reinstalled it fresh.
- Reinstalled it with another version manager (fnm)
- Installed a fresh Angular Project (Without NX) and tests worked, so is not a problem with Angular (Although it was configured with Karma, not jest)
- Installed a fresh Nx Project and failed again.

Migrate the Angular app to Jest instead of Karma, and got new errors, which is always good.

```
wallaby.js started
core v1.0.1324
console.log: Jest: FAIL src/app/app.component.spec.ts

console.log: Jest:   ● Test suite failed to run

    The service is no longer running

      at node_modules/esbuild-wasm/lib/main.js:1381:29
      at sendRequest (node_modules/esbuild-wasm/lib/main.js:662:14)
      at start (node_modules/esbuild-wasm/lib/main.js:1379:9)
      at node_modules/esbuild-wasm/lib/main.js:1924:82


Runtime error: The service is no longer running
    at ./node_modules/esbuild-wasm/lib/main.js:1381:29
    at sendRequest (./node_modules/esbuild-wasm/lib/main.js:662:14)
    at start (./node_modules/esbuild-wasm/lib/main.js:1379:9)
    at ./node_modules/esbuild-wasm/lib/main.js:1924:82
    at FSReqCallback.oncomplete (node:fs:196:23)
```

Does some of this rings a bell?

**Comment by nicolashzmor** [comment]
To keep adding some more weird evidence:

I created this simple config file:
```
module.exports = function (wallaby) {
  return {
    files: [{ pattern: 'tsconfig.spec.json', load: false }],
    tests: ['src/app/**/*.spec.ts'],
    debug: true,
    trace: true,
  }
}
```

And added this tests:
File 1
```typescript
describe('Hello', () => {
  it('should make true', () => {
    expect(true).toEqual(true)
  })
})
```

File 2
```typescript
import { TestBed } from '@angular/core/testing'
import { AppComponent } from './app.component'

describe('AppComponent', () => {
  beforeEach(async () => {
    await TestBed.configureTestingModule({
      declarations: [AppComponent],
    }).compileComponents()
  })

  it('should create the app', () => {
    const fixture = TestBed.createComponent(AppComponent)
    const app = fixture.componentInstance
    expect(app).toBeTruthy()
  })
})
```

This is the console output:
```bash
2022-08-18T17:22:58.188Z workers Sandbox (active) [gb36t] error: TypeError: Failed to resolve module specifier "@angular/core/testing". Relative references must start with either "/", "./", or "../".
Runtime error: TypeError: Failed to resolve module specifier "@angular/core/testing". Relative references must start with either "/", "./", or "../".
2022-08-18T17:22:58.188Z workers Failed to map the stack to user code, entry message: Uncaught TypeError: Failed to resolve module specifier "@angular/core/testing". Relative references must start with either "/", "./", or "../"., stack: TypeError: Failed to resolve module specifier "@angular/core/testing". Relative references must start with either "/", "./", or "../".
2022-08-18T17:22:58.189Z workers [gb36t] Loaded 0 test(s)
2022-08-18T17:22:58.189Z workers [gb36t] Run 0 test(s), skipped 0 test(s)
2022-08-18T17:22:58.190Z workers [gb36t] Sandbox is responsive, closing it
2022-08-18T17:22:58.199Z workers [clshz] Loaded 1 test(s)
2022-08-18T17:22:58.200Z workers [clshz] Test executed: should make true
2022-08-18T17:22:58.217Z workers [clshz] Run 1 test(s), skipped 0 test(s)
2022-08-18T17:22:58.218Z workers [clshz] Sandbox is responsive, closing it
2022-08-18T17:22:58.219Z workers Merging parallel test run results
Finished executing 1 affected test(s) in 215 millisecond(s)
2022-08-18T17:22:58.219Z project Test run finished
2022-08-18T17:22:58.219Z project Processed console.log entries
2022-08-18T17:22:58.219Z project Processed loading sequences
2022-08-18T17:22:58.219Z project Processed executed tests
2022-08-18T17:22:58.220Z project Processed code coverage
2022-08-18T17:22:58.226Z project Test run result processed and sent to IDE
```

**Comment by nicolashzmor** [comment]
@smcenlly I run my project in my macbook instead of Ubuntu and runs without a problem. 

How could we know if the problem is in my setup or Wallaby's? 
Could you test a sample project in latest WebStorm on Ubuntu 22.04?

Thanks in advance for all the help, I think that this details could end up raising where the error is.

**Comment by smcenlly** [comment]
We've managed to reproduce the issue using the latest version of WebStorm on Ubuntu 22.04.

Interestingly we don't get the same problem using VS Code.

From an execution perspective, WebStorm and VS Code are running the launching the exact same Wallaby code.

We *think* it may be related to the WebStorm process not having access to the temp directory that jest uses to cache its transformations.

We'll continue to investigate and let you know what we find.

**Comment by smcenlly** [comment]
When we run `npx jest` from the WebStorm terminal, we get an error. Do you get the same?

```
npx jest             
/home/smcenlly/repos/temp/wallaby-3047/node_modules/fb-watchman/index.js:207
        var error = new Error(obj.error);
                    ^
Error: A non-recoverable condition has triggered.  Watchman needs your help!
The triggering condition was at timestamp=1660867587: opendir(/home/smcenlly/repos/wallaby/test/samples/integration/node-v16/jest/vue-v3-babel-javascript/node_modules/typescript/loc/lcl/PLK/TypeScriptLanguageService) -> The user limit on the total number of inotify watches was reached; increase the fs.inotify.max_user_watches sysctl
All requests will continue to fail with this message until you resolve
the underlying problem.  You will find more information on fixing this at
https://facebook.github.io/watchman/docs/troubleshooting.html#poison-opendir

    at ChildProcess.<anonymous> (/home/smcenlly/repos/temp/wallaby-3047/node_modules/fb-watchman/index.js:207:21)
    at ChildProcess.emit (node:events:527:28)
    at ChildProcess.emit (node:domain:475:12)
    at maybeClose (node:internal/child_process:1092:16)
    at Socket.<anonymous> (node:internal/child_process:451:11)
    at Socket.emit (node:events:527:28)
    at Socket.emit (node:domain:475:12)
    at Pipe.<anonymous> (node:net:709:12) {
  watchmanResponse: {
    version: '2022.06.06.00',
    error: 'A non-recoverable condition has triggered.  Watchman needs your help!\n' +
      'The triggering condition was at timestamp=1660867587: opendir(/home/smcenlly/repos/wallaby/test/samples/integration/node-v16/jest/vue-v3-babel-javascript/node_modules/typescript/loc/lcl/PLK/TypeScriptLanguageService) -> The user limit on the total number of inotify watches was reached; increase the fs.inotify.max_user_watches sysctl\n' +
      'All requests will continue to fail with this message until you resolve\n' +
      'the underlying problem.  You will find more information on fixing this at\n' +
      'https://facebook.github.io/watchman/docs/troubleshooting.html#poison-opendir\n'
  }
}
```

**Comment by smcenlly** [comment]
I suspect that the filesystem watch error we received isn't affecting you as we found another problem after fixing that in our system configuration.

The problem was a little tricky/complicated. When `jest` starts on your project, it transforms some files. One of those transforms spawns a new node process. In Ubuntu by default, node is not part of the `PATH` when WebStorm is launched. Wallaby can be configured to start with a specific version of node, and everything works, but if node is launched by a package without it being in the environment path, it will fail, and in this case was failing silently.

We've updated Wallaby to **prepend** the `node.js` path to your `PATH` environment variable if it is not already a part of your path. This fixes the downstream node dependency issue in your NX project.

Please update to Wallaby core `v1.0.1325`, it will fix your problem.

**Comment by nicolashzmor** [comment]
That was a ride. Thanks for the assistance @smcenlly
